### PR TITLE
Issue-5309: summary graphs added to rollup print

### PR DESCRIPTION
**Pull request by rhernandez-intertech**
connects #5309

### CODE REVIEW

File: src/app/report-rollup/report-summary-graphs/report-summary-pie-chart/report-summary-pie-chart.component.ts [hunk 1]
```diff
@@ -109,5 +115,68 @@ export class ReportSummaryPieChartComponent implements OnInit {
...
+      });
+      textTemplate = '%{label}:<br>%{value:$,.0f}';
+    }
+    else if (this.dataOption == 'carbon') {
+      valuesArr = this.pieChartData.map(dataItem => {
+        return dataItem.carbonEmissions
+      });
+      textTemplate = '%{label}:<br>%{value:,.0f} ' + 'tonne CO<sub>2</sub>';
+    }
+    var data = [{
```
Comment (nbintertech) [inline src/app/report-rollup/report-summary-graphs/report-summary-pie-chart/report-summary-pie-chart.component.ts:151]: Change 'var's to 'let'. The project should always use let for block scoped variables. Most projects will use const as well.

File: src/app/report-rollup/report-summary-graphs/report-summary-graphs.component.ts [hunk 2]
```diff
@@ -59,14 +59,13 @@ export class ReportSummaryGraphsComponent implements OnInit {
     });
   }
 
-  ngOnDestroy() {
-    this.reportSummaryGraphService.clearData();
```
Comment (nbintertech) [inline src/app/report-rollup/report-summary-graphs/report-summary-graphs.component.ts:63]: Why was this removed?

**Review by nbintertech — CHANGES_REQUESTED** [review]
It might just be plotly being laggy, but we might want to open an issue to look at the sidebar x-resizing here. It's really chunky

File: src/app/report-rollup/report-summary-graphs/report-summary-graphs.component.ts (hunk 2 above)
Comment (rhernandez-intertech) [reply src/app/report-rollup/report-summary-graphs/report-summary-graphs.component.ts:63]: Because that method no longer needs to be executed due to the updates done to the configuration of the reportSummaryGraphService

**Review by nbintertech — APPROVED** [review]
